packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -242,10 +253,8 @@ class DateTimePicker extends React.Component {
       value.trim();
```

Comment:
Not exactly related to this PR but `trim()` doesn't modify the string it is called on so this needs to be changed to 
```suggestion
      value = value.trim();
```